StereoKitC/tools/virtual_keyboard.cpp

```diff
@@ -112,10 +110,10 @@ void virtualkeyboard_open(bool32_t open, text_context_ type) {
 
 	// Get the right layout for this text context
 	switch (type) {
-	case text_context_text:  keyboard_active_root = keyboard_ctx_root_text;   break;
-	case text_context_number:keyboard_active_root = keyboard_ctx_root_number; break;
-	case text_context_uri:   keyboard_active_root = keyboard_ctx_root_uri;    break;
-	default:                 keyboard_active_root = keyboard_ctx_root_text;   break;
+	case text_context_text:  keyboard_active_root = text_context_text;   break;
+	case text_context_number:keyboard_active_root = text_context_number; break;
+	case text_context_uri:   keyboard_active_root = text_context_uri;    break;
+	default:                 keyboard_active_root = text_context_text;   break;
 	}
```

Comment:
Does this still need to be a switch/case? This can probably be done much more easily with an `if` now.